.gitignore

```diff
@@ -16,3 +16,4 @@ node_modules_backup
 /.env
 coverage/
 dist
+build
```

Comment:
If this is meant to refer to a directory in the repo root, then `/build/` will be less error-prone.